fe/fe-core/src/test/java/com/starrocks/sql/plan/ReplayFromDumpTest.java

```diff
@@ -63,7 +63,6 @@ public static void beforeClass() throws Exception {
         connectContext = UtFrameUtils.createDefaultCtx();
         connectContext.getSessionVariable().setOptimizerExecuteTimeout(30000);
         connectContext.getSessionVariable().setJoinImplementationMode("auto");
-        connectContext.getSessionVariable().setEnableLocalShuffleAgg(false);
```

Comment:
why delete this

Reply:
This  will effect the cost of shuffle, shoud replay from query dump instead of using fixed value